docs/projects/tad/reporting-standard/0.1a6.md

```diff
@@ -0,0 +1,503 @@
+    version: {version_external_provider}
+references:
+  - {reference_uri}
+interaction_details:
+  - {system_interaction_details}
+version_requirements:
+  - {system_version_requirements}
+deployment_variants:
+  - {system_deployment_variants}
+hardware_requirements:
+  - {system_hardware_requirements}
+product_markings:
+  - {system_product_markings}
+user_interface:
+  - description: {system_user_interface}
+    link: {system_user_interface_uri}
+    snapshot: {system_user_interface_snapshot_uri}
+
+models:
+  - !include {model_card_uri}
+
+assessments:
+  - !include {assessment_card_uri}
+```
+
+### Model Card
+
+```yaml
+provenance:
+  git_commit_hash: {git_commit_hash}
+  timestamp: {modification_timestamp}
+  uri: {modification_uri}
+  author: {modification_author}
+language:
+  - {lang_0}
+license:
+  license_name: {license_name}
+  license_link: {license_uri}
+tags:
+  - {tag_0}
+owners:
+  - oin: {oin}
+    organization: {organization_name}
+    name: {owner_name}
+    email: {owner_email}
+    role: {owner_role}
+
+model-index:
+  - name: {model_id}
+    model: {model_uri}
+    artifacts:
+      - uri: {model_artifact_uri}
+      - content-type: {model_artifact_type}
+      - md5-checksum: {md5_checksum}
+    parameters:
+      - name: {parameter_name}
+        dtype: {parameter_dtype}
+        value: {parameter_value}
+        labels:
+          - name: {label_name}
+            dtype: {label_type}
+            value: {label_value}
+    results:
+      - task:
+          - type: {task_type}
+            name: {task_name}
+        datasets:
+          - type: {dataset_type}
+            name: {dataset_name}
+            split: {split}
+            features:
+              - {feature_name}
+            revision: {dataset_version}
+        metrics:
+          - type: {metric_type}
+            name: {metric_name}
+            dtype: {metric_dtype}
+            value: {metric_value}
+            labels:
+              - name: {label_name}
+                type: {label_type}
+                dtype: {label_type}
+                value: {label_value}
+        measurements:
+          bar_plots:
+            - type: {measurement_type}
+              name: {measurement_name}
+              results:
+                - name: {bar_name}
+                  value: {bar_value}
+          graph_plots:
+            - type: {measurement_type}
+              name: {measurement_name}
+              results:
+                - class: {class_name}
+                  feature: {feature_name}
+                  data:
+                    - x_value: {x_value}
+                      y_value: {y_value}
+```
+
+### Assessment Card
+
+```yaml
+provenance:
+  git_commit_hash: {git_commit_hash}
+  timestamp: {modification_timestamp}
+  uri: {modification_uri}
+  author: {modification_author}
+name: {assessment_name}
+date: {assessment_date}
+contents:
+  - question: {question_text}
+    answer: {answer_text}
+    remarks: {remarks_text}
+    authors:
```

Comment:
It seems urn is missing here, can you add it?

Reply:
It should be missing there because that is the previous version. It is added in latest.md.